test/unit/enricher/identifiers.spec.js

```diff
@@ -188,8 +189,9 @@ describe('IdentifiersEnricher', () => {
     const getCookieStub = sandbox.stub(storage, 'getCookie').throws()
     storage.setCookie(COOKIE_NAME, SIMPLE_COOKIE1)
     const state = { identifiersToResolve: [COOKIE_NAME] }
+    const eventBus = LocalEventBus()
 
-    const resolutionResult = identifiersEnricher.enrich(state, storage)
+    const resolutionResult = identifiersEnricher.enrich(state, storage, eventBus)
```

Comment:
Why are we mixing busses in this case? There is a local one for the enricher and a global one for the external storage.

Reply:
Yes, I wasn't aware of this mixup, I tried to fix it for all cases, hopefully I didn't miss any